packages/mgt-chat/src/utils/rewriteEmojiContent.ts

```diff
@@ -13,11 +13,11 @@ const emojiMatch = (messageContent: string): RegExpMatchArray | null => {
 };
 // iterative repave the emoji custom element with the content of the alt attribute
 // on the emoji element
-const processEmojiContent = (messageContent: string): string => {
+const processEmojiContent = (messageContent: string, replacer = '$2'): string => {
```

Comment:
aren't this function and the rewriteEmojiContent function now dead code?

If there is no code other than the tests calling them then they should be removed.